docs/contributing/feature-stages.md

```diff
@@ -11,11 +14,32 @@ changelog. In most cases, features will only stay in alpha for 1 month.
 We recommend using [GitHub issues](https://github.com/coder/coder/issues) to
 leave feedback and get support for alpha features.
 
+## Early access (EA) features
+
+Early access features are neither feature-complete nor stable. We do not
+recommend using EA features in production deployments.
+
+Coder releases EA features behind an “unsafe” experiment, where they’re
+accessible but not easy to find.
+
+### Early access support expectation
+
+Support for early access features is available through _X_.
+
 ## Experimental features
 
 These features are disabled by default, and not recommended for use in
 production as they may cause performance or stability issues. In most cases,
-features will only stay in experimental for 1-2 weeks of internal testing.
+experimental features are complete, but require further internal testing and
+will only stay in the experimental stage for one or two weeks.
```

Comment:
Let's say one month as it reflects the release cycle.

For full context; We won't release a patch to bring something out of stable, so the shortest amount of time it can be in experimental is 1 month.